### PR TITLE
Delete user

### PR DESCRIPTION
**Pull request by Qtipshooter**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 from flask import Flask
 from flask import render_template, redirect, url_for
 from flask import request, session, flash
+<<<<<<< HEAD
```
Comment (sagarsubedi) [inline app.py:9]: You forgot to remove the merge conflict stuff. It's just these 3 lines

File: static/scripts/script.js [hunk 2]
```diff
@@ -49,30 +49,42 @@ function makeDescriptionHTML(task, taskDesc, isATask=true){
...
+  $.ajax({url:$ACCOUNT_API_PATH, type:"DELETE",
+          data:JSON.stringify(data),
+          contentType:"application/json; charset=utf-8",
+          success:success_function,
+          error:error_function});
+}
+
+//Test function
+function test_success_function(){
+  console.log("TEST SUCCESS")
```
Comment (sagarsubedi) [inline static/scripts/script.js:89]: might wanna remove this as well

**Review by sagarsubedi — APPROVED** [review]
Make sure you do those 2 things, especially the first one.

File: static/scripts/script.js (hunk 2 above)
Comment (Qtipshooter) [reply static/scripts/script.js:89]: Ah yep, forgot that one

File: app.py (hunk 1 above)
Comment (Qtipshooter) [reply app.py:9]: Thats odd.  Will do

**Pull request merged**